packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-groups-claim-dynamic/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Add a groups claim with a dynamic whitelist
+---
+You can use Okta Expression Language Group Functions with dynamic whitelists. Three Group functions help you use dynamic group whitelists: `contains`, `startsWith`, and `endsWith`. These functions return all of the groups that match the specified criteria without needing to have groups specified in the app.
+
```

Comment:
Since "Groups", "User Groups" and "App Groups" are all Okta things, shouldn't they be capitalized?